java-sdk/src/main/java/com/adobe/target/artifact/TargetInvalidArtifactException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package com.adobe.target.artifact;
+
+public class TargetInvalidArtifactException extends RuntimeException {
```

Comment:
I think it would be better if this didn't extend RuntimeException so that it can be properly declared and the developer is forced to catch it.